test/e2e/annotations/customhttperrors.go

```diff
@@ -22,12 +22,17 @@ import (
 
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
-
 	extensions "k8s.io/api/extensions/v1beta1"
 
 	"k8s.io/ingress-nginx/test/e2e/framework"
 )
 
+const defaultBackendName = "upstream-default-backend"
```

Comment:
this is not being used

Reply:
I'll put up a quick follow-up to remove this, wasn't quick enough before merging. thanks for catching it